superset/assets/src/components/TableSelector.jsx

```diff
@@ -370,10 +370,12 @@ export default class TableSelector extends React.PureComponent {
       <div className="section">
         <ControlLabel>
           {t('See table schema')}{' '}
-          <small>
-            ({this.state.tableOptions.length} {t('in')}{' '}
-            <i>{this.props.schema}</i>)
-          </small>
+          {this.props.schema && (
+            <small>
+              ({this.state.tableOptions.length} {t('in')}{' '}
```

Comment:
i know you didn't add it, but this translation block really doesn't do anything here, as translating chunks of sentences doesn't really work. you could fix it if you wanted

```suggestion
              {this.state.tableOptions.length} in{' '}
```